Sources/Jobs/Job+Traversable.swift

```diff
@@ -0,0 +1,38 @@
+import Apodini
+
+/// Extension that defines methods that traverses properties of `Job`s.`
+extension Job {
+    /// Activates all properties of `Job`s at once and always keeps them activated.
+    func activate() -> Self {
+        var selfCopy = self
+
+        apply({ (activatable: inout Activatable) in
+            activatable.activate()
+        }, to: &selfCopy)
+
+        return selfCopy
+    }
+    
+    /// Checks if only valid property wrappers are used with `Job`s.
+    func checkPropertyWrapper() throws {
+        try execute({ (_ :Environment<EnvironmentValues, Connection>) in
+            throw JobErrors.requestPropertyWrapper
+        }, on: self)
+        
+        try execute({ (_: RequestBasedPropertyWrapper) in
+            throw JobErrors.requestPropertyWrapper
+        }, on: self)
+    }
+    
+    /// Subscribes to all `ObservedObject`s.
+    func subscribe(configuration: JobConfiguration) {
+        execute({ (observedObject: AnyObservedObject) in
```

Comment:
Since this function depends on your target-internal `JobConfiguration` it cannot be transferred to the main target as is. I'd suggest to make a public `subscribe<T>(on target: T, using callback: (AnyObservedObject) -> Void)` global in the main target. You can then use this function to implement your extension here.

Reply:
Good idea!

Reply:
Fixed with `c3c4523`